solutions/observability/apps/llm-observability.md

```diff
@@ -0,0 +1,86 @@
+
+## Metrics and logs ingestion for LLM APIs (via Elastic integrations)
+
+Elastic’s LLM integrations now support the most widely adopted models, including OpenAI, Azure OpenAI, and a diverse range of models hosted on Amazon Bedrock and Google Vertex AI. Depending on the LLM provider you choose, the following table shows which source you can use and which type of data -- log or metrics -- you can collect.
+
+| **LLM Provider**  | **Source**  | **Metrics** | **Logs** | 
+|--------|------------|------------|
+| [Amazon Bedrock](https://www.elastic.co/guide/en/integrations/current/aws_bedrock.html)| [AWS CloudWatch Logs][impl-bedrock] | ✅ | ✅ |  
+| [Azure OpenAI](https://www.elastic.co/guide/en/integrations/current/azure_openai.html)| [Azure Monitor and Event Hubs][impl-azure] | ✅ | ✅ | 
+| [GCP Vertex AI](https://www.elastic.co/guide/en/integrations/current/gcp_vertexai.html) | [GCP Cloud Monitoring][impl-vertexai]      | ✅ | 🚧 | 
+| [OpenAI](https://www.elastic.co/guide/en/integrations/current/openai.html) | [OpenAI Usage API][openai-usage] | ✅| 🚧 | 
+
+## APM tracing for LLM models (via instrumentation)
+
+Elastic offers specialized OpenTelemetry Protocol (OTLP) tracing for applications leveraging LLM models hosted on OpenAI, Azure, and Amazon Bedrock, providing a detailed view of request flows. This tracing capability captures critical insights, including the specific models used, request duration, errors encountered, token consumption per request, and the interaction between prompts and responses. Ideal for troubleshooting, APM tracing allows you to find exactly where the issue is happening with precision and efficiency in your LLM-powered application. 
+
+You can instrument the application with one of the following Elastic Distributions of OpenTelemetry (EDOT):
+
+- [Python](https://github.com/elastic/elastic-otel-python)
+- [Node.js](https://github.com/elastic/elastic-otel-node)
+- [Java](https://github.com/elastic/elastic-otel-java)
+
+EDOT includes many types of instrumentation. The following table shows the status of instrumentation relevant to GenAI on a per-language basis:
+
+| **SDK**  | **Language** | **Instrumented Dependency** | **Traces** | **Metrics** | **Logs** | 
+|-------|-----|----|-----|------|------|-----|------|
+| OpenAI | Python | [openai](https://github.com/elastic/elastic-otel-python-instrumentations/tree/main/instrumentation/elastic-opentelemetry-instrumentation-openai)| ✅ | ✅ | ✅ | 
+| OpenAI| JS/Node | [openai](https://github.com/elastic/elastic-otel-node/tree/main/packages/instrumentation-openai) | ✅  | ✅ | ✅ | 
+| OpenAI| Java| [com.openai:openai-java](https://github.com/elastic/elastic-otel-java/tree/main/instrumentation/openai-client-instrumentation) | ✅ | ✅ | ✅| 
+| (AWS) Boto| Python| [botocore](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py)| ✅ | ✅ | ✅ | 
```

Comment:
To be a bit more user friendly we can point to https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/botocore/botocore.html

And I'll take care of updating the content of that README to contain some more content.